setup.py

```diff
@@ -92,6 +92,6 @@ def per_version_requirements(extra_requires_dict):
     params['install_requires'] = parse_requirements('requirements.txt')
     params['test_suite'] = 'unittest.collector'
     params['extras_require']['doc'] = parse_requirements('requirements-docs.txt')
-    per_version_requirements(params)
+    per_version_requirements(params['extras_require'])
```

Comment:
I still think something is not quite right here. Looks like you're not doing anything with the return value.

Reply:
There is no return value : I modify the dictionnary in place. Do you prefere I make something like `params['extras_require'].update(per_version_requirements)`?

Reply:
Yes, more explicit.

Reply:
Or I guess rename to `add_per_version_requirements`

Reply:
done.

Reply:
Haha *or* as in modifying in place is adding requirements to an existing dictionary. A function called `per_version_requiments` I would expect to return something. This is really nitpicking though, you can leave as is.

Reply:
ok